neural_compressor/torch/utils/constants.py

```diff
@@ -26,7 +26,7 @@
         "double_quant_sym": True,
         "double_quant_group_size": 8,
     },
-    "BNB": {
+    "BNB_NF4": {
```

Comment:
please update double quant params. add `use_double_quant` and delete `double_quant_dtype`

Reply:
ok